src/runtime-rs/Makefile

```diff
@@ -320,7 +318,7 @@ ifneq (,$(FCCMD))
     KERNELPATH_FC = $(KERNELDIR)/$(KERNEL_NAME_FC)
     DEFSANDBOXCGROUPONLY_FC = true
     RUNTIMENAME := virt_container
-    DEFSTATICRESOURCEMGMT_FC ?= true
+    DEFSTATICRESOURCEMGMT_FC = true
```

Comment:
Any reason not to switch to simple expansion `:=` like the other ones ?

Reply:
I was wondering about this anomaly as well.

Reply:
the previous ?= -> = was correcting a typo. This one (amended the previous commit) makes is homogeneous across all hypervisors. BTW, TIL the [difference of := and = in Makefiles](https://stackoverflow.com/questions/4879592/whats-the-difference-between-and-in-makefile) ;)